### PR TITLE
Docs: add instructions about chartmuseum

### PR DESCRIPTION
**Pull request by charlie0129**
Signed-off-by: Charlie Chiang <charlie_c_0129@outlook.com>

- Updated `Build your Own Registry` with ChartMuseum and `addon push` command
- Use `addon init` command in `Build Your Own Addon`

Corresponding feature PR: https://github.com/kubevela/kubevela/pull/4261

### CODE REVIEW

File: docs/platform-engineers/addon/addon-registry.md [hunk 1]
```diff
@@ -18,64 +18,90 @@ $ git remote add <repoName> <you git server address>
...
+Helm Chart Repository server.
 
-You can build your own helm repository by following this [tutorial](https://helm.sh/docs/topics/chart_repository/#hosting-chart-repositories).
+In this tutorial, we are going to use [ChartMuseum](https://chartmuseum.com/) to build our repository. If you already have one without ChartMuseum, don't worry. You can still follow these steps, except that you cannot utilize `vela addon push` command and will have to manually upload your addon.
 
-If you already have one, you can follow these steps package add push addon to helm repository.
+Follow the instructions [here](https://chartmuseum.com/#Instructions) to set up your ChartMuseum instance. 
 
-Package the addon:
+> Too complicated to set up? We have got you covered! A ChartMuseum addon is on the way.
```
Comment (wonderflow) [inline docs/platform-engineers/addon/addon-registry.md]: haha great!

File: docs/platform-engineers/addon/intro.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ The picture below shows what KubeVela does when an addon is enabled. There are m
 
 To build an addon, you should follow some basic rules as follows.
 
-You need to create an addon directory to place addon resource files. We're building a [CLI command](https://github.com/kubevela/kubevela/pull/4162) to create the scaffold, it will be released in v1.5.
+You need to create an addon directory to place addon resource files. Won't bother to create? `vela addon init` command will create a basic structure for you. Please refer to `vela addon init -h` for details. For quick starts, we will just use `vela addon init your-addon-name` now.
```
Comment (wonderflow) [inline docs/platform-engineers/addon/intro.md]: you need to add the cli version required

**Review by wonderflow — COMMENTED** [review]
does this doc for 1.5 only?

**Comment by charlie0129** [comment]
> does this doc for 1.5 only?

Yes, 1.5 or later

**Review by wonderflow — APPROVED** [review]
lgtm. But Please hold the merge until 1.5 released

File: docs/reference/addons/chartmuseum.md [hunk 3]
```diff
@@ -0,0 +1,241 @@
+---
+title: cert-manager
```
Comment (wonderflow) [inline docs/reference/addons/chartmuseum.md]:
```suggestion
title: ChartMuseum
```

File: docs/platform-engineers/addon/addon-registry.md [hunk 4]
```diff
@@ -18,57 +18,119 @@ $ git remote add <repoName> <you git server address>
...
-vela addon registry add my-repo --type helm --endpoint=<URL> --username=<username> --password=<passwor>
+```shell
+$ vela addon registry list
+Name    	Type	URL                        
+...
+localcm 	helm	http://localhost:8080 
+```
+
+### Push an addon to your registry
+
```
Comment (wonderflow) [inline docs/platform-engineers/addon/addon-registry.md:86]:
```suggestion

> Note: you need to upgrade your CLI to v1.5.0+ for this feature.

```

**Pull request merged**